app/models/nic/bmc.rb

```diff
@@ -1,7 +1,14 @@
 module Nic
   class BMC < Managed
     PROVIDERS = %w(IPMI)
+    before_validation :ensure_physical
     validates :provider, :presence => true, :inclusion => { :in => PROVIDERS }
+    validates :mac, :presence => true, :if => :managed?
+
+    def virtual
```

Comment:
Log this behavior and comment if really needed, although I'd really prefer if we get to the root of the issue and fix why `Nic::BMC.create/new` is being called with `virtual = true`.